### PR TITLE
Outline of data fetching from NGI sources

### PR DESCRIPTION
**Pull request by alneberg**
I promise to change branch name after this. 😂

### CODE REVIEW

File: daily_read/ngi_data.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    def __init__(self, config, sources):
+        self.config = config
+        self.sources = sources
+        self.source_names = [source.name for source in sources]
+        self.__data_fetched = False
+
+    def get_data(self):
+        for source in self.sources:
+            try:
+                source.get_data()
```
Comment (aanil) [inline daily_read/ngi_data.py:18]: Will this write to the git repo? Or will that be implemented in the child(?) class?

File: daily_read/ngi_data.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        return ["orderportal_id2", "orderportal_id3"]
+
+    def stage_data_for_project(self, project_id):
+        pass
+
+    def commit_staged_data(self, message):
+        pass
+
+
+class StockholmProjectData(object):
```
Comment (aanil) [inline daily_read/ngi_data.py:58]: Should this inherit from `ProjectDataMaster`?

**Review by aanil — APPROVED** [review]
Just a few questions, but I understand its a work in progress

File: daily_read/ngi_data.py (hunk 2 above)
Comment (aanil) [reply daily_read/ngi_data.py:58]: Only asking because it's mentioned as a subclass

Comment (alneberg) [reply daily_read/ngi_data.py:58]: No, just removed that mentioning

File: daily_read/ngi_data.py (hunk 1 above)
Comment (alneberg) [reply daily_read/ngi_data.py:18]: This will write to the git repo yes

**Pull request merged**